ui/app/AppLayouts/Communities/controls/TokenHolderListItem.qml

```diff
@@ -178,7 +179,7 @@ ItemDelegate {
 
                 TokenHolderNumberCell {
                     Layout.alignment: Qt.AlignRight
-                    text: LocaleUtils.numberToLocaleString(root.amount)
+                    text: LocaleUtils.numberToLocaleString(StatusQUtils.AmountsArithmetic.toNumber(root.amount, root.multiplierIndex))
```

Comment:
I would opt for using `amout` directly here:
```suggestion
                    text: amount
```
to make this component more general and do the formatting in the component that's using it (SortableTokenHoldersList) in this case. And no need to pass `multiplerIndex` then.